File: biometric/src/main/java/dev/skomlach/biometric/compat/utils/hardware/HardwareInfo.java
package dev.skomlach.biometric.compat.utils.hardware;

import androidx.annotation.RestrictTo;

@RestrictTo(RestrictTo.Scope.LIBRARY)
public interface HardwareInfo {

    boolean isHardwareAvailable();

    boolean isBiometricEnrolled();

    boolean isLockedOut();
}

File: biometric/src/main/java/dev/skomlach/biometric/compat/utils/hardware/HardwareInfo.kt
package dev.skomlach.biometric.compat.utils.hardware

import androidx.annotation.RestrictTo

@RestrictTo(RestrictTo.Scope.LIBRARY)
interface HardwareInfo {
    val isHardwareAvailable: Boolean
    val isBiometricEnrolled: Boolean
    val isLockedOut: Boolean
}